paddle/fluid/framework/details/op_graph_view.h

```diff
@@ -65,6 +65,7 @@ bool OpGraphView::VisitAllPendingOps(OpHandleBase *op,
         if (!callback(pending_op)) {
           return false;
         }
```

Comment:
Prefer "while" loop than "do while" loop. 

There is q.push(op) before the while loop. It's no harm to write
while (!q.empty()) {
    ...
}

Reply:
Done.